src/lib/components/NewLayeredMap/NewLayeredMap.jsx

```diff
@@ -338,6 +339,11 @@ NewLayeredMap.propTypes = {
         position: PropTypes.string,
     }),
 
+    /**
+     * UnitScale is a box that displays the the current unit scale on the map.
+     */
+    unitScale: PropTypes.bool,
+
```

Comment:
sorry, I changed this. Should be like this:
```suggestion
    /**
     * UnitScale is a box that displays the the current unit scale on the map.
     */
    unitScale: PropTypes.shape({
        position: PropTypes.string,
    }),

```